### PR TITLE
Policy management feature

### LINKED ISSUE #8: Support for zone / folder and policy creation
**BUSINESS PROBLEM**
We are trying to automate onboarding

**PROPOSED SOLUTION**
Support creation of zones / folders and the associated attributes with this provider.

**CURRENT ALTERNATIVES**
scripts, etc.

**VENAFI EXPERIENCE**
4 months, every day

### PR DESCRIPTION
**Pull request by angelmoo**
now using terreform-provider-venafi, user can create on VaaS or TPP and can import a policy from VaaS or TPP
and also test cases for creating and importing policies were implemented.

### CODE REVIEW

**Review by marcos-albornoz — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,5 @@ condor.tf
 terraform-provider-venafi
 credentials
 chain.pem
-!*/**
\ No newline at end of file
+!*/**
+terraform.d
```
Comment (rvelaVenafi) [inline .gitignore]: No new line at end of file

File: test_files/empty_policy.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+
+}
```
Comment (rvelaVenafi) [inline test_files/empty_policy.json]: No new line at end of file

File: test_files/policy_specification_cloud.json [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    },
+    "keyPair": {
+      "keyType": "RSA",
+      "rsaKeySize": 2048,
+      "ellipticCurve": "",
+      "generationType": "",
+      "serviceGenerated": true
+    }
+  }
+}
```
Comment (rvelaVenafi) [inline test_files/policy_specification_cloud.json]: No new line at end of file

File: test_files/policy_specification_tpp.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      "state": "Utah",
+      "country": "US"
+    },
+    "keyPair": {
+      "keyType": "RSA",
+      "rsaKeySize": 3072,
+      "ellipticCurve": ""
+    }
+  }
+}
```
Comment (rvelaVenafi) [inline test_files/policy_specification_tpp.json]: No new line at end of file

File: test_files/policy_specification_tpp_management.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+      "country": "US"
+    },
+    "keyPair": {
+      "keyType": "RSA",
+      "rsaKeySize": 2048,
+      "serviceGenerated": true
+    },
+    "autoInstalled": false
+  }
+}
```
Comment (rvelaVenafi) [inline test_files/policy_specification_tpp_management.json]: No new line at end of file

**Review by rvelaVenafi — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (angelmoo) [reply .gitignore]: fixed on the most recent commit

File: test_files/empty_policy.json (hunk 2 above)
Comment (angelmoo) [reply test_files/empty_policy.json]: fixed on the most recent commit

File: test_files/policy_specification_tpp.json (hunk 4 above)
Comment (angelmoo) [reply test_files/policy_specification_tpp.json]: fixed on the most recent commit

File: test_files/policy_specification_tpp_management.json (hunk 5 above)
Comment (angelmoo) [reply test_files/policy_specification_tpp_management.json]: fixed on the most recent commit

File: test_files/policy_specification_cloud.json (hunk 3 above)
Comment (angelmoo) [reply test_files/policy_specification_cloud.json]: fixed on the most recent commit

**Review by marcos-albornoz — APPROVED** [review]

**Review by tr1ck3r — APPROVED** [review]

**Review by rvelaVenafi — APPROVED** [review]

**Pull request merged**